### PR TITLE
CY-1758 Allow any db cluster node count

### PR DESCRIPTION
**Pull request by geokala**

### CODE REVIEW

File: cfy_manager/components/postgresql_server/postgresql_server.py [hunk 1]
```diff
@@ -675,14 +710,14 @@ def _determine_cluster_status(self, db_nodes):
                 )
             )
             status = max(status, self.DOWN)
-        if etcd_followers < 1:
+        if etcd_followers < majority_requirement:
             logger.error(
                 'No etcd followers found, cluster consensus lost.'
```
Comment (tehasdf) [inline cfy_manager/components/postgresql_server/postgresql_server.py]: that is no longer true

Comment (geokala) [reply cfy_manager/components/postgresql_server/postgresql_server.py]: In what situation is it not true?

Comment (geokala) [reply cfy_manager/components/postgresql_server/postgresql_server.py]: If you're talking about the 1 node case, that'll result in majority_requirement = 1 // 2, which is 0.

Comment (geokala) [reply cfy_manager/components/postgresql_server/postgresql_server.py]: OK, a quick out-of-band discussion clarified: The message itself was wrong. Updated.

**Pull request merged**